src/app/logging.ts

```diff
@@ -56,8 +60,9 @@ if (process.env.NODE_ENV === 'development') {
 }
 
 export const logger =
+    process.env.DISABLE_LOGGING_AND_METRICS === 'true'
+        ? getLogger('off')
+        :
     process.env.NODE_ENV === 'development'
         ? getLogger('development')
         : getLogger();
```

Comment:
Linter not happy, maybe splitting it out something like this makes sense?

```suggestion
const loggerOff = process.env.DISABLE_LOGGING_AND_METRICS === 'true' && 'off';
const loggerDev = process.env.NODE_ENV === 'development' && 'development';

// If Disabled, turn off logger. If in Dev, set logger to development.
export const logger = getLogger(loggerOff || loggerDev || undefined);
```

Reply:
Thanks for the suggestion! I've seen a similar construct to set default values for variables, but it's the first time I'm seeing this one. Learned something!

There was also another nested tertiary operator that I had to untangle at the top of the file. In doing that I ended up splitting the initialisation of log4js, which then changed this part of the code. Have a look, see what you think!